### PR TITLE
Enable JeOS-for-kvm-and-xen-sdboot flavor

### PR DESCRIPTION
**Pull request by rbmarliere**
This PR contains some small fixes required to run the new Tumbleweed JeOS-for-kvm-and-xen-sdboot flavor.

- Related ticket: https://progress.opensuse.org/issues/169354
- Needles: https://github.com/os-autoinst/os-autoinst-needles-opensuse/pull/823
- OBS trigger: https://github.com/os-autoinst/openqa-trigger-from-obs/pull/282
- Schedule: https://github.com/os-autoinst/opensuse-jobgroups/pull/572
- Verification run: https://rmarliere-openqa.qe.prg2.suse.org/tests/1098

### CODE REVIEW

**Review by linguini-dev — APPROVED** [review]
LGTM

File: tests/jeos/firstrun.pm [hunk 1]
```diff
@@ -93,7 +93,7 @@ sub verify_norepos {
 sub verify_bsc {
     if (is_qemu && is_x86_64 && script_run("rpm -q qemu-guest-agent") != 0) {
         # Included in SLE-15-SP2+, TW and Leap
-        die("bsc#1207135 - Missing qemu-guest-agent from virtual images") unless is_sle('<15-SP2');
+        die("bsc#1207135 - Missing qemu-guest-agent from virtual images") unless is_sle('<15-SP2') || is_bootloader_sdboot;
```
Comment (mloviska) [inline tests/jeos/firstrun.pm:96]: How is this related to systemd boot? If the package is in other TW images, aren't we covering a bug?

Comment (rbmarliere) [reply tests/jeos/firstrun.pm:96]: Thanks for pointing that out. I wasn't sure about this one, but the fact is that qemu-guest-agent does not come installed in the JeOS-for-kvm-and-xen-sdboot image. The sub verify_norepos is only ran if is_jeos, so I thought adding that unless wouldn't be harmful. Please advise :)

Comment (mloviska) [reply tests/jeos/firstrun.pm:96]: I would go for a bug report as the qemu-ga is recommended to be used in any qemu VM.

Comment (rbmarliere) [reply tests/jeos/firstrun.pm]: Will do! As for this PR, I removed the change in https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/20804/commits/fa5412fa1d25cdf3f42de73cabf567502a857d34

**Review by mloviska — APPROVED** [review]

File: tests/jeos/firstrun.pm (hunk 1 above)
Comment (rbmarliere) [reply tests/jeos/firstrun.pm]: https://bugzilla.opensuse.org/show_bug.cgi?id=1234610

**Review by grisu48 — APPROVED** [review]

**Review by ricardobranco777 — APPROVED** [review]

File: tests/installation/bootloader_uefi.pm [hunk 2]
```diff
@@ -72,6 +72,11 @@ sub run {
         }
     }
 
+    if (is_jeos && get_required_var('FLAVOR') =~ /sdboot/) {
```
Comment (Vogtinator) [inline tests/installation/bootloader_uefi.pm:75]: Should be just `is_bootloader_sdboot`
